### PR TITLE
use builtin contract addresses for preprod/prod

### PR DESCRIPTION
**Pull request by bflood-strike**

### CODE REVIEW

File: Censo.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -1552,14 +1556,14 @@
...
 				MTL_FAST_MATH = YES;
-				NEW_SETTING = "";
 				ONLY_ACTIVE_ARCH = YES;
 				OTHER_SWIFT_FLAGS = "-Xfrontend -debug-time-function-bodies";
 				RAYGUN_API_KEY = CtOnGQjIo1U8dELkoUf0iw;
 				RAYGUN_ENABLED = no;
 				SDKROOT = iphoneos;
 				SWIFT_ACTIVE_COMPILATION_CONDITIONS = DEBUG;
 				SWIFT_OPTIMIZATION_LEVEL = "-Onone";
+				USE_BUILTIN_CONTRACT_ADDRESSES = no;
```
Comment (bholzman) [inline Censo.xcodeproj/project.pbxproj:1566]: Suggest calling this “USE_STATIC_CONTRACT_ADDRESSES” ?

File: Censo/EvmConfig.swift [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+struct EvmConfig {
+    static let censoGuard = "CensoGuard"
+    static let censoTransfersOnlyGuard = "CensoTransfersOnlyGuard"
+    static let censoTransfersOnlyWhitelistingGuard = "CensoTransfersOnlyWhitelistingGuard"
+    static let censoWhitelistingGuard = "CensoWhitelistingGuard"
+    
+    static let ethereumGuardAddresses = Configuration.useBuiltInContractAddress ? [
+        ContractNameAndAddress(name: censoGuard, address: "0x42313af5c7b79c89b1892c24d28afe44a5ea522f", deprecated: false),
+        ContractNameAndAddress(name: censoGuard, address: "0x196dd5959460f4654e4ba1e5196546884e79d060", deprecated: true),
```
Comment (bholzman) [inline Censo/EvmConfig.swift:18]: I think it’s a great idea to have this structure to allow for deprecated contracts, but wondering if we need to include the pre-production contracts in here — maybe we can just drop support for older contracts?

**Review by bholzman — APPROVED** [review]
Looks good, but maybe we should talk with everyone about including the deprecated contracts or not

File: Censo.xcodeproj/project.pbxproj (hunk 1 above)
Comment (bflood-strike) [reply Censo.xcodeproj/project.pbxproj]: renamed

File: Censo/EvmConfig.swift (hunk 2 above)
Comment (bflood-strike) [reply Censo/EvmConfig.swift]: removed the deprecated guards

**Pull request merged**